### ISSUE TITLE
Email signature editor not accessible in Outlook 2013

### ISSUE THREAD

**Issue by mjanusauskas** [nvaccess/nvda#5890]
NVDA does not seem to be able to access or review the email signature editor area in the Options > Email > Signatures area of Outlook 2013.

**Comment by Laughingthunder** [comment]
I can confirm this.

**Comment by Cambridgeport90** [comment]
So can I … not sure about 2016 .. will look when I get home.

**Comment by wallaceid** [comment]
I can confirm for Outlook 2016 as well

**Comment by comanna** [comment]
Hi, 
I can confirm this for Outlook 2016 with both NVDA 2016.3 and JAWS 2016. Could it be that MS have a new wiziwig editor here?

All the best,

cearbhall

**Comment by sucharugupta** [comment]
I can confirm this to happen with Outlook 2016, While tabbing to this field there is no output by NVDA.

**Comment by AlexandreMooc** [comment]
I can confirm this in Outlook 2016 with JAWS 2016.2.1. Signature is not read upon tabbing into the signature text input field, and running the cursor through the text with arrow key generates not output either.

**Comment by jcsteh** [comment]
This was apparently fixed in #3833 (merge 7fceb1d, notable commit 297b2ffe). However, it either regressed or it was never really fixed in the first place. See also #4102 concerning `AppModule.isUIAWindow` and #6005 which will need a similar fix according to @MichaelDCurran.

**Comment by leonardder** [comment]
What is the current state of this? Is it still broken? I belief I've been able to review and update my signature just fine with Outlook 2016.

**Comment by michaelDCurran** [comment]
The signature editor in Outlook is implemented as a Word document, 
however provides no actual way of getting to the object model.
For Office 2010 and below we could sort of use displayModel to get some 
information. For Office 2010 we use UIA. Office 2013 can provide 
neither. They no longer use GDI, but UIA has not yet been properly 
implemented.
Thus, as far as I can tell we cannot fix this.
